build.xml

```diff
@@ -190,9 +217,17 @@
         <string>--add-opens java.base/jdk.internal.module=ALL-UNNAMED</string>
         <string>--add-opens java.base/jdk.internal.util.jar=ALL-UNNAMED</string>
         <string>--add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED</string>
+
+        <string>--add-opens java.base/sun.nio.ch=ALL-UNNAMED</string>
```

Comment:
Are those justified? I was looking for ways not to add openings.

Reply:
Perhaps we can add these on the simulator launch path only, to avoid taking on new add-opens just to unblock it.